### PR TITLE
tutufnjksdjfsdjk

### PR DESCRIPTION
**Pull request by Rzeps0n**
desdfwfwer

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -492,8 +492,5 @@ I throughly recommend these resources to continue your Git practice:
 Author
 ------
 
-This work is licensed under the Creative Commons
-Attribution-NonCommercial-ShareAlike 3.0 License\
-<a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/3.0/">http://creativecommons.org/licenses/by-nc-sa/3.0/</a>\
-Author: Thong Kuah\
-Contributors: Andy Newport, Nick Malcolm
```
Comment (Rzeps0n) [inline README.markdown:499]: demo

this is illegal, and bs.
keep that in

File: nowy_plik [hunk 2]
```diff
@@ -0,0 +1 @@
+hello, world!
```
Comment (Rzeps0n) [inline nowy_plik:1]: add quotes

Comment (GrzybuGrzybu) [inline nowy_plik:1]: add print

**Review by Rzeps0n — COMMENTED** [review]
apply changes

File: nowy_plik (hunk 2 above)
Comment (Rzeps0n) [reply nowy_plik]: done!

File: README.markdown (hunk 1 above)
Comment (Rzeps0n) [reply README.markdown]: done!

File: nowy_plik (hunk 2 above)
Comment (Rzeps0n) [reply nowy_plik]: done!

Comment (GrzybuGrzybu) [reply nowy_plik]: :D

**Pull request merged**